pkg/command/global.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"sync/atomic"
+)
+
+var atomicInt int32
+
+// SetGlobalVerbose sets the global command verbosity to the specified value
```

Comment:
Nice use of atomic bool here! Do you think it is worth mentioning in a note why this is used and also that `atomicInt` should never be accessed outside the context of these functions? It shouldn't be too much of a problem as it is not an exported var and the `command` pkg is pretty scoped, but it could cause a data race if used incorrectly.

Reply:
Sure thing, I added some commend and rebased on top of the latest master to resolve the merge conflicts.